app/views/devise/confirmations/new.html.erb

```diff
@@ -5,7 +5,7 @@
 
   <div class="field">
     <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true %>
+    <%= f.email_field :email, required: true, readonly: true, input_html: { value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) } %> 
```

Comment:
Well, we don't do that for any of the other forms I think. So if we are doing those changes, it needs to be a more complete overhaul (or users can just use simple form :D).